bundles/tools.vitruv.applications.uml.testutility/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Vitruv UML Utils
+Bundle-SymbolicName: tools.vitruv.applications.uml.testutility
+Automatic-Module-Name: tools.vitruv.applications.uml.testutility
+Bundle-Version: 0.3.0.qualifier
+Require-Bundle: tools.vitruv.testutils.vsum,
+ org.junit.jupiter.api,
+ tools.vitruv.applications.pcmumlclass,
+ tools.vitruv.applications.util.temporary,
+ org.eclipse.xtend.lib,
+ edu.kit.ipd.sdq.commons.util.java,
+ edu.kit.ipd.sdq.activextendannotations;bundle-version="1.5.0"
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Export-Package: tools.vitruv.applications.uml.testutility.uml
```

Comment:
Having uml in the package declaration twice looks a bit odd to me?

Reply:
Refactored package in [4896351](https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/48963512de89ee8b878a7503f83e4ddaa1e2114f)